high_performance/modules/high_performance/include/klepsydra/high_performance/event_loop.h

```diff
@@ -70,27 +70,31 @@ class EventLoop
     EventLoop(std::shared_ptr<EventEmitterInterface<EventloopDataWrapper>> &externalEventEmitter,
               RingBuffer &ringBuffer,
               const std::string &name,
-              long timeoutUS = EVENT_LOOP_START_TIMEOUT_MICROSEC)
+              const long timeoutUS = EVENT_LOOP_START_TIMEOUT_MICROSEC,
+              const std::vector<int> &cpuAffinity = {})
         : _name(name)
         , _threadName(std::to_string(BufferSize) + "_" + name)
         , _ringBuffer(ringBuffer)
         , _eventHandler(externalEventEmitter)
         , _isStarted(false)
         , _eventLoopTask([this] {
+            spdlog::debug("kpsr::high_perf::EventLoop::_eventLoopTask begin");
```

Comment:
```suggestion
            spdlog::trace("kpsr::high_perf::EventLoop::_eventLoopTask begin");
```

Reply:
Same for end, ported to kpe-core.
`spdlog::debug(EVENT_LOOP_START_MESSAGE)` kept.